Sources/Quick/NSBundle+CurrentTestBundle.swift

```diff
@@ -11,6 +11,16 @@ extension Bundle {
         return allBundles.first { $0.bundlePath.hasSuffix(".xctest") }
     }
 
+    /**
+     Return the module name of the bundle.
+     Uses the bundle filename and transform it to match Xcode's transformation.
+     Module name has to be a valid "C99 extended identifier".
+     */
+    internal var moduleName: String {
+        return bundleURL.fileName
+            .replacingOccurrences(of: " ", with: "_")
+            .replacingOccurrences(of: "-", with: "_")
+    }
```

Comment:
Can you reuse [the same mechanism as the one we use for test names](https://github.com/Quick/Quick/blob/0fdf273c7c1463a5f25c3942587452b1e892918c/Sources/QuickObjectiveC/NSString%2BQCKSelectorName.m#L5-L34) here? Or is there a reason we can't?